doc/devel/coding_style_guideline.rst

```diff
@@ -7,17 +7,13 @@ DIPY Coding Style Guideline
 The main principles behind DIPY_ development are:
 
 * **Robustness**: the results of a piece of code must be verified
-  systematically, and hence stability and robustness of the code must be
-  ensured, reducing code redundancies.
+  systematically, and hence stability and robustness of the code must be ensured, reducing code redundancies.
 * **Readability**: the code is written and read by humans, and it is read
   much more frequently than it is written.
 * **Consistency**: following these guidelines will ease reading the code,
   and will make it less error-prone.
 * **Documentation**: document the code. Documentation is essential as it is
-  one of the key points for the adoption of DIPY as the toolkit of choice in
-  diffusion by the scientific community. Documenting helps clarifying
-  certain choices, helps avoiding obscure places, and is a way to allow
-  other members *decode* it with less effort.
+  one of the key points for the adoption of DIPY as the toolkit of choice in diffusion by the scientific community. Documenting helps to clarify certain choices, helps to avoid obscure places, and is a way to allow other members *decode* it with less effort.
```

Comment:
I would break this up a bit and streamline a tiny bit, like so:

```suggestion
  one of the key points for the adoption of DIPY as the toolkit of choice in 
  diffusion by the scientific community. Documenting helps to clarify certain 
  choices, helps to avoid obscure places, allowing other members 
  to *decode* it with less effort.
```

Reply:
Okay sure will revert that